WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormViewModelTests.swift

```diff
@@ -0,0 +1,118 @@
+import Photos
```

Comment:
nit: this import seems not needed 👍

Reply:
indeed! removed in https://github.com/woocommerce/woocommerce-ios/pull/2846/commits/19396ff0470831b2f357f600d2619d1f8373dfe5